.circleci/config.yml

```diff
@@ -597,6 +597,7 @@ workflows:
               only:
                 - master
                 - develop
+                - s3-test-fixes
```

Comment:
why is this branch being added here?

Reply:
I had added it for testing the fixes. Removed now